### DISCUSSION TITLE
Getting Errors While running tests

### DISCUSSION THREAD

**Discussion by hardud** [idaholab/moose#28426] [Q&A Getting Started] [answered]
### Check these boxes if you have followed the posting rules.

- [X] I have consulted the [Posting Guidelines](https://github.com/idaholab/moose/discussions/18270).
- [X] I have searched the [Discussions Forum](https://github.com/idaholab/moose/discussions) and [MOOSE Framework Troubleshooting](https://mooseframework.inl.gov/help/troubleshooting.html) and have not found what I was looking for
- [X] Q&A Getting Started is the most appropriate category for my question (trouble installing, beginner user, ...)

### Issue or question about MOOSE

Everything was running fine until yesterday. When I tried running a regression test today, I am getting this error: symbol not found in flat namespace '__ZN5neml25Model12set_solutionERKNS_11BatchTensorE'

Make command is showing no errors, but while running tests, they are failing and I see the following errors:
<img width="956" alt="Screenshot 2024-08-19 at 9 32 00 AM" src="https://github.com/user-attachments/assets/7081d9ab-6591-4b39-8ae4-302a8c3689c4">

In case it helps solving the errors, during the start of the day, I tried getting Neml2 with the following commands:
cd projects/
git clone git@github.com:reverand/neml2.git
cd moose/
export NEML2_DIR=$HOME/projects/neml2

Please let me know how should I get rid of the errors.

**Comment by GiudGiud** [comment, marked as answer]
Hello

Save all your work with git add / git commit
then `git clean -xfd` in moose/
then recompile

these symbol not found is probably because an old compiled object is looking for a routine that does not exist anymore, or at least not with this signature

Guillaume

**Reply by hardud** [reply]
Thanks, that worked:)

**Comment by milljm** [comment]
>Everything was running fine until yesterday

Usually tells us, that you performed an update of some kind. Be it, involving `conda update/install` and/or `git fetch/pull` commands.

Basically what you want to do, is start over. That is; `git clean -xfd`'s etc, and rebuild.

Edit:
I am referring to our [Staying Up To Date](https://mooseframework.inl.gov/getting_started/new_users.html) section on our Wiki.

**Reply by hardud** [reply]
Thank you:) That was helpful
